.changeset/two-mayflies-tell.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/integration': patch
```

Comment:
This is a 1.x package so a new non-breaking feature should be `minor`.

Should probably make a separate changeset for this one too! Saying like "Added X and Y" which gives sufficient context.